x-pack/plugins/apm/server/routes/service_groups/get_service_group_alerts.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { kqlQuery } from '@kbn/observability-plugin/server';
+import {
+  AggregationsCardinalityAggregate,
+  AggregationsFilterAggregate,
+  QueryDslQueryContainer,
+} from '@elastic/elasticsearch/lib/api/types';
+import { Logger } from '@kbn/core/server';
+import { ApmPluginRequestHandlerContext } from '../typings';
+import { SavedServiceGroup } from '../../../common/service_groups';
+import { ApmAlertsClient } from './get_apm_alerts_client';
+import {
+  KIBANA_ALERT_RULE_PRODUCER,
+  KIBANA_ALERT_STATUS,
+} from '../../../common/rules/apm_rule_types';
```

Comment:
Could we just use `ALERT_RULE_PRODUCER` and `ALERT_STATUS` from `@kbn/rule-data-utils`